plat/kvm/arm/entry64.S

```diff
@@ -40,6 +40,7 @@
  * paging_init (it invalidates our stack)
  */
 .section .bss
+.align 16
```

Comment:
I see that, unlike x86, we do not have an explicit aligning of the stack like the `andq    $~0xf, %rsp`. Would it be worth adding this to ARM as well, just to double make sure.

Reply:
Based on offline discussion, updated PR to:
* Remove changes in x86_64, the alignment is enforced on runtime
* Update arm64 to enforce alignment on runtime too